quisp/modules/ConnectionManager.cc

```diff
@@ -231,35 +151,31 @@ static int fill_path_division(std::vector<int> path /**< Nodes on the connection
  *
  * \todo Where should timeouts and error handling happen?
  **/
-void ConnectionManager::intermediate_alloc_res_handler(ConnectionSetupResponse *pk) {
-  // here return ack?
+void ConnectionManager::storeRuleSet(ConnectionSetupResponse *pk) {
```

Comment:
The naming `foo_bar_handler` confuses us. we should give it the name that describes what the method does.